front/app/src/Component/ConnectionState/DisplayAnonymousConnection.tsx

```diff
@@ -0,0 +1,247 @@
+			}}
+			open={server.isFetching}
+		>
+			<Alert
+				severity={props.severity}
+			>
+				{
+					(server.error)
+					? <></>
+					: loading
+				}
+				<Typography
+					sx={
+					{
+						mt: 1,
+						fontSize: "0.7rem"
+					}}
+				>
+					{props.message}
+				</Typography>
+				<Typography
+					sx={
+					{
+						mt: 1,
+						fontSize: "0.5rem"
+					}}
+				>
+					{props.descriptMessage}
+				</Typography>
+			</Alert>
+		</Backdrop>
+	);
+};
+
+const	DisplayAnonymousConnection = () =>
+{
+	const	dispatch = useAppDispatch();
+	const	registrationStep = useAppSelector((state) =>
+	{
+		return (state.anonymousUser.registrationStep);
+	});
+	const	anonymousUser = useAppSelector((state) =>
+	{
+		return (state.anonymousUser);
+	});
+
+	let		message;
+	let		descriptMessage;
+	let		severity: AlertColor | undefined;
+
+	switch (registrationStep)
+	{
+		case "undefined":
+			severity = "info";
+			message = "";
+			descriptMessage = "Initialisation de la session";
+			break ;
+		case "reset":
+			severity = "error";
+			message = "Error";
+			descriptMessage = "Reinitialisation de la page";
+			break ;
+		case "uuid-creation-started":
+			severity = "info";
+			message = "";
+			descriptMessage = "Generation de votre identifiant";
+			break ;
+		case "uuid-creation-sucess":
+			severity = "success";
+			message = "";
+			descriptMessage = "Identifiant generes";
+			break ;
+		case "register-anonymous-started":
+			severity = "info";
+			message = "";
+			descriptMessage = "Enregistrement sur le serveur";
+			break ;
+		case "register-anonymous-success":
+			severity = "success";
+			message = "";
+			descriptMessage = "Enregistrement reussi aupres du serveur";
+			break ;
+		case "login-anonymous-started":
+			severity = "info";
+			message = "";
+			descriptMessage = "Request login to the server ";
+			break ;
+		case "login-anonymous-success":
+			severity = "success";
+			message = "";
+			descriptMessage = "Session Active";
+			break ;
+		case "token-verification-started":
+			severity = "info";
+			message = "";
+			descriptMessage = "Verify you session token";
+			break ;
+		case "token-verification-success":
+			severity = "success";
+			message = "";
+			descriptMessage = "Your session token is valid";
+			break ;
+		default:
+			severity= "warning";
+			message = "";
+			descriptMessage = "If message persist please contact dev team";
+			break ;
+	}
+
+	// console.info(anonymousUser);
+
+	// logic of registration
+	useEffect(() =>
+	{
+		// create the user unique identifier
+		if (registrationStep === "undefined")
+		{
+			if (anonymousUser.uuid !== NIL)
+			{
+				// stored value tested to trigger:
+				// eslint-disable-next-line max-len
+				// {"registrationStep":"\"undefined\"","uuid":"\"161e3594-be9d-4afd-bdfe-d21aa7128350\"","creationDate":"\"\"","password":"\"\"","message":"\"\"","expireAt":"-1","token":"\"\"","error":"false","errorMessage":"\"\"","errorStatusCode":"0","_persist":"{\"version\":-1,\"rehydrated\":true}"}
```

Comment:
Is this commented line an example ?

Reply:
This is the state of the app when need a state reset.
I keep it for testing the reset step from front side by copying this object into localstorage.

Reply:
So is it needed on master? Or is it something you should have added with ``git add -p`` to keep the state of the file as is on your local branch but ditch the commented lines from the commit ?